app/src/App.js

```diff
@@ -1,216 +1,22 @@
-          <br />
-          <input
-            type="radio"
-            id="age"
-            name="age"
-            value="18-35"
-            onChange={(e) => setAge(e.target.value)}
-          />{" "}
-          18-35
-          <br />
-          <input
-            type="radio"
-            id="age"
-            name="age"
-            value="35+"
-            onChange={(e) => setAge(e.target.value)}
-          />{" "}
-          35+
-        </label>
-        <br />
-        <label>
-          Message
-          <br />
-          <textarea
-            name="message"
-            value={message}
-            onChange={(e) => setMessage(e.target.value)}
-          ></textarea>
-        </label>
-        <br />
-        <br />
-      </form>
-      <button onClick={() => subForm()}>Submit</button>
-    </>
-  );
-};
-
-//Formik
-
-const SignupForm = () => {
-  const formik = useFormik({
-    initialValues: {
-      firstName: "",
-      lastName: "",
-      email: "",
-    },
-    validationSchema: Yup.object({
-      firstName: Yup.string()
-        .max(15, "Must be 15 characters or less")
-        .required("Required"),
-      lastName: Yup.string()
-        .max(20, "Must be 20 characters or less")
-        .required("Required"),
-      email: Yup.string().email("Invalid email address").required("Required"),
-    }),
-    onSubmit: (values) => {
-      alert(JSON.stringify(values, null, 2));
-    },
-  });
-
-  return (
-    <>
-      <h1>Formik Form</h1>
-      <form onSubmit={formik.handleSubmit}>
-        {/* Pass the useFormik() hook initial form values and a submit function that will
-      be called when the form is submitted */}
-        <label htmlFor="firstName">First Name</label>
-        <input
-          id="firstName"
-          name="firstName"
-          type="text"
-          onChange={formik.handleChange}
-          onBlur={formik.handleBlur}
-          value={formik.values.firstName}
-        />
-        {formik.touched.firstName && formik.errors.firstName ? (
-          <div>{formik.errors.firstName}</div>
-        ) : null}
-
-        <label htmlFor="lastName">Last Name</label>
-        <input
-          id="lastName"
-          name="lastName"
-          type="text"
-          onChange={formik.handleChange}
-          onBlur={formik.handleBlur}
-          value={formik.values.lastName}
-        />
-        {formik.touched.lastName && formik.errors.lastName ? (
-          <div>{formik.errors.lastName}</div>
-        ) : null}
-
-        <label htmlFor="email">Email Address</label>
-        <input
-          id="email"
-          name="email"
-          type="email"
-          onChange={formik.handleChange}
-          onBlur={formik.handleBlur}
-          value={formik.values.email}
-        />
-        {formik.touched.email && formik.errors.email ? (
-          <div>{formik.errors.email}</div>
-        ) : null}
-        <br />
-        <button type="submit">Submit</button>
-      </form>
+      <Router>
```

Comment:
If you plan on having the `<Nav />` on all pages, one idea is to stick it here instead! You'd put it right above `<Router>` ( [ex](https://github.com/carbonsoda/scribbler/blob/main/app/src/App.js#L22)).

Reply:
Ty for this suggestion!!